layouts/partials/example-js-grpc.html

```diff
@@ -12,14 +12,16 @@
 async function main() {
   const clientStub = new dgraph.DgraphClientStub("localhost:9080", grpc.credentials.createInsecure());
   const dgraphClient = new dgraph.DgraphClient(clientStub);
+
   const query = `<span class="query-content">{{ .Code }}</span>`;
-{{ if eq .Vars nil }}
+  {{- if eq .Vars nil }}
```

Comment:
Here and in js-http, are these purely whitespace / code style change?
If so those should be done alongside meaningful changes to corresponding files (in order to reduce the noise in commit logs)